samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs

```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
 {
     class Program
     {
-        static IQueueClient queueClient;
+        static IQueueClient _queueClient;
 
         static void Main(string[] args)
         {
-            string ServiceBusConnectionString = "";
-            string QueueName = "";
+            string serviceBusConnectionString = string.Empty;
+            string queueName = string.Empty;
 
-            for (int i = 0; i < args.Length; i++)
+            for (var i = 0; i < args.Length; i++)
             {
-                var p = new Program();
-                if (args[i] == "-ConnectionString")
+                switch (args[i])
                 {
-                    Console.WriteLine($"ConnectionString: {args[i+1]}");
-                    ServiceBusConnectionString = args[i + 1]; // Alternatively enter your connection string here.
+                    case "-ConnectionString":
+                        Console.WriteLine($"ConnectionString: {args[i+1]}");
+                        serviceBusConnectionString = args[i + 1]; // Alternatively enter your connection string here.
+                        break;
+                    case "-QueueName":
```

Comment:
👍

Reply:
I thought about going one step further and using a new 8.0 switch expression here, but didn't want to bewilder newer devs